### DISCUSSION TITLE
Request

### DISCUSSION THREAD

**Discussion by GholibjonMadiyarov** [dbeaver/dbeaver#19592] [Q&A] [answered]
Navigate between tab panels (Tables) using the key Ctrl+Tab

**Comment by E1izabeth** [comment, marked as answer]
DBeaver has already got the keys for navigation between tabs 
![image](https://user-images.githubusercontent.com/28875055/229527990-7d41aebe-319a-42d8-b7fd-85792a3551ef.png)


You can change hotkeys for these actions in preferences.
![image](https://user-images.githubusercontent.com/28875055/229527855-82f63357-f6f1-472d-acc1-00c845de1af4.png)
